rodan/urls.py

```diff
@@ -235,7 +237,8 @@
         name="resource-viewer-acquire",
     ),
     url(r"^api/labels/$", ResourceLabelList.as_view(), name="resourcelabel-list"),
-    url(r"^api/label/(?P<pk>[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})/$",
+    url(
+        r"^api/label/(?P<pk>[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})/$",
```

Comment:
This is unrelated to archives but ok.

Reply:
This is just what was making the linter mad earlier.